assets/scss/_footer.scss

```diff
@@ -0,0 +1,32 @@
+.footer{
+    background-image: url("../images/footerbg.svg");
+    background-size: cover;
+    background-repeat: no-repeat;
```

Comment:
you can shorten it to one line: `background: url("../images/footerbg.svg") cover no-repeat;`

https://developer.mozilla.org/en-US/docs/Web/CSS/background

Reply:
done